handbook/technical-overview/auth/System.md

```diff
@@ -0,0 +1,5 @@
+# Kerfislýsingar
+
+[System Manual](system/system-manual.md)
+
+[**Service Provider Manual**](system/service-provider-manual.md)
```

Comment:
Does this need to be bold as the file contains only two links, is this more important or relevant link than the one above?